Show checkouted branch (#4080)
zkSNACKs/WalletWasabi: WalletWasabi.Documentation/BackendDeployment.md

## Prompt

> Additionally to the welcome banner it could be good to know in what server we are all the time and see clearly what branch is checkout-ed, in this case update the prompt as follow:

Reviewer: ```suggestion
Additionally to the welcome banner it could be good to know in which server we are all the time and to see clearly which branch is checked out, in this case update the prompt as follow:
```
